src/main/java/liuni/resources/LiUniResource.java

```diff
@@ -13,6 +13,7 @@
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
```

Comment:
be consistent and either use ResponseBuilder or don't throughout the entire file